draft-ietf-webtrans-overview.md

```diff
@@ -222,7 +222,28 @@ attacks without an explicit indication from the client.
 All transport protocols MUST provide datagrams, unidirectional and
 bidirectional streams in order to make the transport protocols interchangeable.
 
-## Datagrams
+## Session-Wide Features  {#features-session}
+
+Any WebTransport protocol SHALL provide the following operations on the
+session:
+
+establish a session
+: Create a new WebTransport session given a URI {{!RFC3986}} and the origin
+  {{!RFC6454}}.
+
+terminate a session
+: Terminate the session while communicating to the peer an unsigned 32-bit
+  error code and an error reason string of at most 1024 bytes.  The error code
+  and string are optional; the default values are 0 and "".
+
+Any WebTransport protocol SHALL provide the following events:
+
+session terminated event
+: Indicates that the WebTransport session has been terminated, and no user data
+  can be exchanged on it any further.  Similarly to the "terminate a session"
+  operation above, an error code and an error string can be provided.
```

Comment:
This is not yet used in the W3C spec, but the path migration event will be useful to update the max datagram size.

Reply:
Looking through the notes for w3c/webtransport#314, I think we should probably leave this as-is for now.  If we do end up adding that, I would prefer us to go for something more specific like "max datagram size changed event", since this makes it easier for the API user to have the correct expectations on how this event is used.